ci_framework/playbooks/07-deploy-edpm-with-bmaas.yml

```diff
@@ -0,0 +1,46 @@
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Call make_edpm_compute_baremetal target
+      ansible.builtin.include_role:
+        name: 'install_yamls_makes'
+        tasks_from: 'make_edpm_compute_baremetal'
+      vars:
+        make_edpm_compute_baremetal_env:
+          NETWORK_ISOLATION: false
+          OPENSTACK_IMG: "{{ cifmw_operator_build_output['operators']['openstack-operator']['image_catalog'] }}"
+
+    - name: Generate Bash Script to check if compute nodes are provisioned
+      ansible.builtin.copy:
```

Comment:
Can we parameterize the timeouts here? today 600s is enough, but you know how vexx is awesome with high load.

Reply:
@viroel thanks for the suggestion, I will parameterize the timeout in the next patch set.

Reply:
@viroel I have parameterized the timeout in role.